### ISSUE TITLE
remove() doesn't actually remove child view controller

### ISSUE THREAD

**Issue by vkaramov** [justeat/ScrollingStackViewController#20]
remove() doesn't actually remove child view controller. To check that try to replace 

```swift
        if isHiding {
            
            hideShowButton.title = "Show 5"
            hide(viewController: self.segments[4])
        } else {
```

with

```swift
        if isHiding {
            
            hideShowButton.title = "Show 5"
            remove(viewController: self.segments[4])
        } else {
```

in the sample App.

**Comment by vkaramov** [comment]
P.S. Happens on iOS12

**Comment by ghost** [comment]
Hi @vkaramov, we think to have solved the issue with #23. Version 4.0.1 has now been published. 😉
